### PR TITLE
[docs] Add status header for activedirectoryds receiver

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for activedirectoryds, pls review :)

### CODE REVIEW

File: receiver/activedirectorydsreceiver/README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 # Active Directory Domain Services Receiver
 
-The `active_directory_ds` receiver scrapes metric relating to an Active Directory domain controller using the Windows Performance Counters.
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
```
Comment (codeboten) [inline receiver/activedirectorydsreceiver/README.md:7]: This component is not currently in any distributions

**Review by codeboten — COMMENTED** [review]
Pinging @djaglowski @binaryfissiongames for the stability of this component.

File: receiver/activedirectorydsreceiver/README.md (hunk 1 above)
Comment (Frapschen) [reply receiver/activedirectorydsreceiver/README.md]: ok.delete

**Review by BinaryFissionGames — APPROVED** [review]
Beta stability is reasonable for this component.

**Review by codeboten — APPROVED** [review]

**Pull request merged**